docs/vector_store.ipynb

```diff
@@ -419,6 +445,27 @@
     "await store.aapply_vector_index(index)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `ScaNN` index creation (specific to AlloyDB Omni) requires sufficient maintenance work memory. You need to set the database flag `maintenance_work_mem` by calling `set_maintenance_work_mem` before applying the index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg.indexes import ScaNNIndex\n",
+    "\n",
+    "VECTOR_SIZE = 768  # Replace with the vector size of your embedding model\n",
+    "index = ScaNNIndex()\n",
+    "await omni_store.set_maintenance_work_mem(index.num_leaves, VECTOR_SIZE)\n",
+    "await omni_store.aapply_vector_index(index)"
```

Comment:
Let's use "store" instead of "omni_store", so they are easy copy/pasteable.

Reply:
updated